src/test-utils/dom/prompt-input/index.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { act, Simulate } from 'react-dom/test-utils';
+import { ComponentWrapper, ElementWrapper, usesDom } from '@cloudscape-design/test-utils-core/dom';
+import styles from '../../../prompt-input/styles.selectors.js';
+
+export default class PromptInputWrapper extends ComponentWrapper {
+  static rootSelector = styles.root;
+
+  findNativeTextarea(): ElementWrapper<HTMLTextAreaElement> {
+    return this.find<HTMLTextAreaElement>(`.${styles.textarea}`)!;
+  }
+
+  findSubmitButton(): ElementWrapper<HTMLButtonElement> {
```

Comment:
suggest calling this `findActionButton` as that's how it's referred to in all docs